core/src/main/java/org/mule/client/DefaultLocalMuleClient.java

```diff
@@ -268,7 +269,7 @@ public MuleContext getMuleContext()
 
         public MessageInfoMapping getMessageInfoMapping()
         {
-            return null;
+            return MuleMessageInfoMapping.INSTANCE;
```

Comment:
I was not talking about the missing private constructor bu why are you using a singleton instead of creating a new instance